### PR TITLE
ECOS-10673 Dummy conversion / ECOS-11055 implementation

### PR DESCRIPTION
**Pull request by palenikm**
Fix ECOS-11081 (AddPars folder isn't copied from the source folder) and ECOS-11086 (The end of the text is cut off above the path of the destination folder)

### CODE REVIEW

File: RINT/AddOn.grc [hunk 1]
```diff
@@ -37,7 +37,7 @@
 /* [ 22] */ TextEdit			  330    55  380   23	LargePlain    readOnly      1024
 /* [ 23] */ Button				  716    55   90   23	LargePlain    "Browse"
 
-/* [ 24] */ LeftText			  330   101  276   18	LargePlain    "Destination folder for converted favorite XML files and logs:"
+/* [ 24] */ LeftText			  330   101  300   18	LargePlain    "Destination folder for converted favorite XML files and logs:"
```
Comment (ak-nagy) [inline RINT/AddOn.grc:40]: I think we should change this value for the 21st item too.

File: Src/FCConverter.cpp [hunk 2]
```diff
@@ -39,14 +39,23 @@ IO::Folder Converter::GetUniqueDestinationFolder (const GS::UniString& parentFol
...
+}
+
 
 void Converter::CopyToDestinationFolder ()
 {
     auto copyOne = [&] (const IO::Name& name, bool isFolder) {
-        if (isFolder || name.GetExtension () != "xml") {
-            return;
+        if (IsAddParsFolder (name, isFolder) || IsXMLFile (name, isFolder)) {
+            sourceFolder.Copy (name, destinationFolder, name);
```
Comment (ak-nagy) [inline Src/FCConverter.cpp:57]: Do the copy works recursively for folders? Did you check it?

**Review by ak-nagy — CHANGES_REQUESTED** [review]
Please change the 21st item's width too.

File: Src/FCConverter.cpp (hunk 2 above)
Comment (palenikm) [reply Src/FCConverter.cpp:57]: Yes it does! Yes I did!

**Review by ak-nagy — APPROVED** [review]

**Review by ak-nagy — APPROVED** [review]

**Pull request merged**